topics/xamarin/index.md

```diff
@@ -0,0 +1,20 @@
+---
+aliases: mono
+created_by: Alexandru Stefan Popescu
```

Comment:
This should display the authors of Xamarin (`Miguel de Icaza, Nat Friedman`), not your name